rero_ils/modules/collections/api.py

```diff
@@ -70,8 +70,14 @@ def get_items(self):
         :param self: self
         :return: list of items linked to collection
         """
-        return [Item.get_record_by_pid(item['pid']) for
-                item in self.replace_refs().get('items', [])]
+        items = []
+        for item in self.replace_refs().get('items', []):
+            item = Item.get_record_by_pid(item['pid'])
+            # inherit holdings call number when possible
+            if item.issue_inherited_first_call_number:
+                item['call_number'] = item.issue_inherited_first_call_number
```

Comment:
Using this, you call two times the property and so potentially load two times the holdings (perf).
But, I don't think you need to check if `item.issue_inherit_first_call_number` is none, you can remove the if statement and directly assign `item.issue_inherit_first_call_number` to `item['call_number']`.

Same remark for others files.